app/src/main/java/com/github/sdp/ratemyepfl/utils/MapActivityUtils.kt

```diff
@@ -7,6 +7,8 @@ object MapActivityUtils {
         "Arcadie" to R.raw.arcadie,
         "Epicure" to R.raw.epicure,
         "Niki" to R.raw.niki,
-        "Roulotte du Soleil" to R.raw.soleil_roulotte
+        "Roulotte du Soleil" to R.raw.soleil_roulotte,

```

Comment:
We should put these photos in the Firebase Storage and fetch them when needed.

Reply:
Because adding a ressource in our repo for each restaurant/event is not scalable.

Reply:
Yes for sure, this was a temporary solution I wrote before we used Firebase Storage. For next sprint I have several fixes to make about the use of Storage.